geospaas/catalog/tests.py

```diff
@@ -175,8 +175,7 @@ def test_unique_constraint_geographic_location(self):
 
     def test__reproduce__not_null_constraint_failed(self):
         geom_wkt = 'POLYGON((1.9952502916543926 43.079137301616434,1.8083614437225106 43.110281163194,1.6280391132319614 43.13999933989308,1.4543047771860391 43.168322409488,1.287178802518546 43.19527992537942,1.126680477150093 43.22090040126175,0.9728280404789855 43.24521129666272,0.8256387132257121 43.26823900332679,0.6851287265540695 43.2900088324148,0.5513133503959514 43.31054500249216,0.4177032107533156 43.3308546554019,0.4177032107533156 43.3308546554019,0.31209072545607186 42.966172534807384,0.2072770834059167 42.60138984322352,0.10324224766647609 42.23651548835664,-3.327518831779395e-05 41.87155835974214,-0.10256845388361147 41.50652732885059,-0.20438175048840848 41.14143124914533,-0.305491137731497 40.776278956093606,-0.4059141156224018 40.4110792671334,-0.5056677274094622 40.045840981598744,-0.6188735003262834 39.62838980058282,-0.6188735003262834 39.62838980058282,-0.4938090620192412 39.60834071737128,-0.36846516623385345 39.58812662484392,-0.2367679070115216 39.566753658618175,-0.09872965092928164 39.54419980869909,0.045636463325510676 39.520442055142105,0.19631650129236156 39.49545637806485,0.35329570832956364 39.469217768317954,0.516558484513175 39.441700238839005,0.686088358898322 39.412876836714965,0.8618679632011015 39.382719655976956,0.8618679632011015 39.382719655976956,0.985334472893415 39.799577386800905,1.0941941665822539 40.164279112775866,1.2038450353475123 40.52892574316672,1.3143064728956748 40.89350812553239,1.425598388091744 41.25801708090206,1.5377412226553768 41.622443403572326,1.6507559695776892 41.98677786085107,1.764664192292795 42.351011192745254,1.8794880446399878 42.71513411159017,1.9952502916543926 43.079137301616434))'
-        with self.assertRaises(django.db.utils.IntegrityError):
-            gg, created = GeographicLocation.objects.get_or_create(geometry=WKTReader().read(geom_wkt))
+        gg, created = GeographicLocation.objects.get_or_create(geometry=WKTReader().read(geom_wkt))
```

Comment:
If the error due to max precision does not happen anymore, this test can probably be deleted.

Or maybe it still happens with a particular database backend?
Even if it is the case, I am not sure if we should keep that test, since it does not validate a desired behavior.
Maybe it would be more relevant to create an issue about this problem, describing steps to reproduce it.

Any thoughts about that @akorosov, @mortenwh ?

Reply:
I haven't seen it myself.

Reply:
done in 04899cf